content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb

```diff
@@ -79,24 +79,22 @@ def self.remote_command(task, transformation_host, command, stdin = nil, run_as
 
             def self.virtv2vwrapper_options_vmwarews2rhevm_vddk(task)
               source_vm = task.source
-              source_ems = source_vm.ext_management_system
               source_cluster = source_vm.ems_cluster
 
               destination_cluster = task.transformation_destination(source_cluster)
               destination_ems = destination_cluster.ext_management_system
               destination_storage = task.transformation_destination(source_vm.hardware.disks.select { |d| d.device_type == 'disk' }.first.storage)
 
-              vmware_uri = "vpx://"
-              vmware_uri += "#{source_ems.authentication_userid.gsub('@', '%40')}@#{source_ems.hostname}/"
-              vmware_uri += "#{source_cluster.v_parent_datacenter.gsub(' ', '%20')}/#{source_cluster.name.gsub(' ', '%20')}/#{source_vm.host.uid_ems}"
+              vmware_uri = "esx://"
+              vmware_uri += "root@#{source_vm.host.ipaddress}/"
```

Comment:
Again, this might be wrong if it's an IPv6 address.  Suggest you create URIs using the URI's build method
https://github.com/ManageIQ/manageiq/pull/13932